src/shared/components/oncoprint/ResultsViewOncoprint.tsx

```diff
@@ -1793,6 +1882,50 @@ export default class ResultsViewOncoprint extends React.Component<
         return WindowStore.size.width - 75;
     }
 
+    @action.bound
+    public handleClinicalTrackColorChange(
+        value: string,
+        color: RGBAColor | undefined
+    ) {
+        setClinicalTrackColor(
```

Comment:
we should not need to update two different things with essentially the same data.  if we update
store, then the track _should_  respond to the observable mutation.  because of the particular nature of the oncopring library, this may be difficult.